src/animation.js

```diff
@@ -54,6 +69,8 @@ export function extractAnimatedPropValues(props) {
 }
 
 class Animation extends PureComponent {
+  static ANIMATED_SERIES_PROPS = ANIMATED_SERIES_PROPS;
```

Comment:
Seemed like having this as a static property on `Animation` instead of a util exported from `series-utils` was a cleaner API for access from other packages.

Reply:
This looks exciting!

FWIW, in deck.gl etc we are also introducing multiple npm modules to allow users to select the pieces that they need, but we are stil keeping all the code inside a single github repository (them in separate source trees). We prefer this approach for a couple of reasons. In particular, we feel that there is a significant overhead in keeping build tooling, scripts, techniques etc consistent across multiple repos.